bumpversion/vcs.py

```diff
@@ -7,14 +7,21 @@
 import subprocess
 from tempfile import NamedTemporaryFile
 
-from bumpversion.exceptions import WorkingDirectoryIsDirtyException, MercurialDoesNotSupportSignedTagsException
+from bumpversion.exceptions import (
+    WorkingDirectoryIsDirtyException,
+    MercurialDoesNotSupportSignedTagsException
```

Comment:
Personally I'm a fan of trailing commas since it makes the diff smaller when you're adding another import.